doc/api/stream.md

```diff
@@ -1797,6 +1797,10 @@ class SourceWrapper extends Readable {
 *Note*: The `readable.push()` method is intended be called only by Readable
 Implementers, and only from within the `readable._read()` method.
 
+For streams not operating in object mode, if the `chunk` parameter of
+`readable.push()` is `undefined`, it will be treated as empty string or
+buffer.
```

Comment:
What would mean for the `chunk` to be interpreted as an empty string of buffer?
Can you add another sentence explaining what is happening?

Reply:
@MoonBall ping

Reply:
@mcollina [`readable.push('')`](https://nodejs.org/dist/latest-v9.x/docs/api/stream.html#stream_readable_push)

Reply:
@mcollina I added a sentence for [`readable.push('')`](https://nodejs.org/dist/latest-v9.x/docs/api/stream.html#stream_readable_push).